feature/about/src/main/java/io/github/droidkaigi/confsched2019/about/ui/item/AboutSection.kt

```diff
@@ -46,10 +48,16 @@ class AboutSection @Inject constructor(
                 },
                 AboutItem(
                     name = R.string.about_app_version,
-                    description = R.string.about_version_name,
+                    descriptionString = activity.getVersionName(),
                     isLektonCheckText = true
                 )
             )
         )
     }
+
+    private fun Context.getVersionName() = try {
+        packageManager.getPackageInfo(packageName, 0).versionName
```

Comment:
instead of getting from `PackageManager`, what about [adding it to strings using `resValue`](https://www.tanelikorri.com/tutorial/android/set-variables-in-build-gradle/)?

Reply:
That great!
Thank you for advice!!